Requirements.md

```diff
@@ -69,6 +70,41 @@ Game Marketplace:
 - A section of the platform where users can discover and playtest new games.
 - Option for developers to monetize games through in-game purchases, ads, or premium versions.
 
+### 2.4 Events
+
+General Events:
+
+- Events can be created by any logged-in user on the platform at any time.
+- Users can submit events for evaluation, including workshops, lectures, or other activities related to game development.
+- Once approved, events are available for the community to view, attend, and participate in.
+
+Elected Meetings:
+
+- Every week, the community votes on one event to become the "Elected Meeting," a highlighted event in the weekly calendar.
```

Comment:
Don't use "every week", this puts pressure on us. rework this phrase to make it a routine but not every week.